tests/utils_tests/test_autoreload.py

```diff
@@ -252,3 +252,32 @@ def test_resets_trans_real(self):
         self.assertEqual(trans_real._translations, {})
         self.assertIsNone(trans_real._default)
         self.assertIsInstance(trans_real._active, _thread._local)
+
+
+class RestartWithReloaderTests(SimpleTestCase):
+
+    def test_restart_with_reloader_with_manage_py(self):
```

Comment:
I think "restart_with_reloader" in the test name is redundant with the class name. Also, since this test is already passing without the fix, it can be added in a separate commit that precedes the bug fix.